app/views/bots/_form.html.erb

```diff
@@ -15,8 +15,29 @@
     <%= f.label :name %><br/>
     <%= f.text_field :name, :class => "form-control" %>
   </div><br/>
+  <div class="field">
+      <%= f.label :collaborators %>
+      <ul>
+		<% User.with_role(:collaborator, bot).each do |user| %>
+			<p><%= user.username %> <%= link_to 'Remove', "/bots/#{bot.id}/collaborators/#{user.id}", :method => :delete, :data => { :confirm => "Are you sure you want to do this?" } %></p>
```

Comment:
Generally, prefer using `url_for` or a path/url helper rather than hardcoding paths, because it means you don't have to change the code if you change a route. You probably want `url_for(controller: :bots, action: :remove_collaborator, bot_id: bot.id, collaborator: user.id)` here.